sparse/impl/KokkosSparse_spadd_symbolic_impl.hpp

```diff
@@ -589,6 +590,7 @@ void spadd_symbolic_impl(
   // provide the number of NNZ in C to user through handle
   size_type cmax;
   Kokkos::deep_copy(exec, cmax, Kokkos::subview(c_rowmap, nrows));
+  exec.fence("fence before cmax used on host");
```

Comment:
Can do the same with this one too, declare ``cmax`` up on 574 and then pass it to the ``kk_exclusive_parallel_prefix_sum``

Reply:
👍

Reply:
Yeah that's fine